doc/developer_guide/design.md

```diff
@@ -139,7 +139,23 @@ We write integration tests for the VS in Java:
 * `+` Using Maven allows generating build scripts etc. with `project-keeper`
 * `-` Mixing of two languages in a single project
 
-### Integration Tests Using Dynamic Junit Jupiter Tests
+### Convert Table and Column Names to Upper Case
+
+VSDAB converts Databricks table and columns to upper case.
+
+Rationale: Exasol converts unquoted identifiers to upper case. If a table or column contains lower case characters, users need to specify the exact case and quote the identifier in double quotes `"`. This is inconvenient and a potential source of errors.
+
+Converting all names to upper case introduces the risk of conflicts, e.g. when a Databricks table contains columns with the same name but different case, e.g. `COL` and `col`. However this is a problem, because creating such tables in Databricks is not possible and fails with the following error:
```

Comment:
```suggestion
Converting all names to upper case introduces the risk of conflicts, e.g. when a Databricks table contains columns with the same name but different case, e.g. `COL` and `col`. However this is not a problem, because creating such tables in Databricks is not possible and fails with the following error:
```

Reply:
I understand that there is no risk of having conflicts because you cannot create columns `col` and `COL`. But maybe I misunderstood the exaplanation....